src/flags.ts

```diff
@@ -78,6 +78,14 @@ const open = (options = {}): Parser.flags.IBooleanFlag<boolean> => {
   });
 };
 
+const breaking = (options = {}): Parser.flags.IBooleanFlag<boolean> => {
+  return flags.boolean({
+    char: 'F',
```

Comment:
What is the purpose of this char `'F'` ? How is it used ?

Reply:
How I understand, this is the shortcut for `--breaking` flag. And since `-B` is already for `--branch`, you had to select an other letter. `F` for `Force, right ?

Reply:
yes and `f` is already taken by the `format` flag.